deployments/ansible/molecule/custom_vars/converge.yml

```diff
@@ -23,4 +23,4 @@
   tasks:
     - name: "Include signalfx.splunk_otel_collector.collector role"
       include_role:
-        name: "collector"
+        name: "signalfx.splunk_otel_collector.collector"
```

Comment:
LGTM. Another option would be to add the path to roles explicitly like below under the [ansible provisioner config](https://github.com/signalfx/splunk-otel-collector/blob/main/deployments/ansible/molecule/config/docker.yml#L21). Maybe add the path and keep the full name both to cover all bases?
```
  env:
    ANSIBLE_ROLES_PATH: ../../roles
 ```
 
Probably not related but another thing I noticed is the [role_name in galaxy_info](https://github.com/signalfx/splunk-otel-collector/blob/main/deployments/ansible/roles/collector/meta/main.yml#L3) is incorrect. I wonder if this is just a leftover from a legacy role.

Reply:
Giving a spin to `ANSIBLE_ROLES_PATH` it seems a good addition to make the tests consistent between the platforms - now I have to understand why there is a vagrant.yml...

Regarding role_name on the galaxy file, per schema at https://raw.githubusercontent.com/ansible/ansible-lint/main/src/ansiblelint/schemas/meta.json it doesn't allow the `.` so keeping as it is, at least for now.